test/test_torch.py

```diff
@@ -4446,7 +4468,7 @@ def test_unbind(self):
 
     def test_linspace(self):
         devices = ['cpu'] if not torch.cuda.is_available() else ['cpu', 'cuda']
-        for device in devices:
+        for _device in devices:
```

Comment:
why do we change device to _device? this breaks the CI. I have created a fix: https://github.com/pytorch/pytorch/pull/17995
Let's whether it's enough or not

If not, let's revert this change.